client/src/lifecycleManager.js

```diff
@@ -29,16 +29,18 @@ class LifecycleManager extends LuigiClientBase {
 
     /**
      * Iterates over an object and executes all top-level functions
-     * with a given payload.
+     * with a given payload and the Luigi core domain
      * @private
      */
-    const _callAllFns = (objWithFns, payload) => {
```

Comment:
Please note that once [this pr](https://github.com/SAP/luigi/pull/635/files#diff-18560a5dd187d564cad88d5a36754bce) is merged we will need to update the changes done to this function.